draft-ietf-quic-recovery.md

```diff
@@ -1133,8 +1133,8 @@ limits and so no advantage is gained by doing so.
 
 Endpoints choose the congestion controller that they use. Congestion controllers
 respond to reports of ECN-CE by reducing their rate, but the response may vary.
-Markings can be treated as equivalent to loss ({{?RFC3168}}), but other
-responses can be specified, such as ({{?RFC8511}}) or ({{?RFC8311}}).
+Markings can be treated as equivalent to loss ({{!RFC3168}}), but other
```

Comment:
This section doesn't have normative language, so I don't see a need for these references to be normative.